src/lib/components/pages/TenantNewEntity.vue

```diff
@@ -1,11 +1,22 @@
 <template>
-  <TenantHome title="New Entity" :breadcrumb-links="breadcrumbLinks" :errors="errors">
+  <TenantHome :title="this.parentId?'New Child Entity':'New Entity'" :breadcrumb-links="breadcrumbLinks" :errors="errors">
```

Comment:
Move this `title` to a computed function. Also, based on that the breadcrumb texts also can be changed.

Reply:
parentId is the computed function, the title is just the ternary operation. I don't think it's better to add the title to the computed function it uses unnecessary cache memory.

Reply:
I meant like this.
https://github.com/apache/airavata-custos-portal/blob/master/src/lib/components/pages/ListTenants.vue#L132

Reply:
I understand that, but all the computed functions are stored in the cache. `title` is used at only one place unlike `parentId`, making `title` as a computed function will consume cache moreover it has 2 options.

Reply:
I'm keeping the condition as part of HTML, I don't feel like adding it into the cache for the single call/use purpose.